### PR TITLE
Add product metadata to every page

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Tests are failing because:

- `Mapbox Terrain-RGB` should be changed to `Mapbox Terrain-RGB tileset` for consistency with other similar products
- `Mapbox Satellite tileset` should be added to the list of valid product names

### Final checklist

- [x] Run `npm install @mapbox/remark-lint-mapbox@1.0.0-beta.1`
- [x] Add `products` to each markdown file
- [x] Ask for `products` review
- [x] Update `npm install @mapbox/remark-lint-mapbox@latest`
- [x] Update docs-page-shell (manually) from https://github.com/mapbox/docs-page-shell/pull/78 (Run `npm ci`, then `npm run build`, copy `dist/page-shell-script.js` and overwrite the repo's `vendor/docs-page-shell/page-shell-script.js` file.)
- [x] Update `src/components/page-shell.js` using code snippets below.
- [x] Test locally (any browser) to make sure events come through in [Segment](https://app.segment.com/mapbox/sources/mapbox-staging/debugger)
- [x] Test in IE 11 to make sure events come through in [Segment](https://app.segment.com/mapbox/sources/mapbox-staging/debugger)
- [x] Push to staging to make sure events come through in [Segment](https://app.segment.com/mapbox/sources/mapbox-staging/debugger)
- [x] Update from publisher-production
- [ ] Merge!

cc @domlet @katydecorah

### CODE REVIEW

File: docs/pages/examples.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: 'Mapbox GL JS Examples'
 description: 'Code examples for Mapbox GL JS.'
 product: maps
 pathname: /mapbox-gl-js/examples/
+products:
+- Documentation
```
Comment (domlet) [inline docs/pages/examples.js:7]:
```suggestion
- Mapbox GL JS
```

Comment (domlet) [reply docs/pages/examples.js:7]: What do you think of using `Documentation` as product only when the products are generated, unpredictable, or ambiguous?

File: docs/pages/plugins.js [hunk 2]
```diff
@@ -2,6 +2,8 @@
 title: Plugins
 description: Extend your Mapbox GL JS map with plugins.
 pathname: /mapbox-gl-js/plugins/
+products:
+- Documentation
```
Comment (domlet) [inline docs/pages/plugins.js:6]:
```suggestion
- Mapbox GL JS
```

**Comment by katydecorah** [comment]
This is the metadata for each page:

```json
{
  "/mapbox-gl-js/examples/": {
    "product": "Documentation",
    "platform": "",
    "service": "platform"
  },
  "/mapbox-gl-js/api/geography/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "API"
  },
  "/mapbox-gl-js/api/events/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "API"
  },
  "/mapbox-gl-js/api/handlers/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "API"
  },
  "/mapbox-gl-js/api/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "API"
  },
  "/mapbox-gl-js/api/map/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "API"
  },
  "/mapbox-gl-js/api/markers/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "API"
  },
  "/mapbox-gl-js/api/sources/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "API"
  },
  "/mapbox-gl-js/api/properties/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "API"
  },
  "/mapbox-gl-js/example/3d-buildings/": {
    "product": "Mapbox GL JS,Mapbox Streets tileset",
    "platform": "web",
    "service": "maps,data services",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/3d-extrusion-floorplan/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/add-3d-model/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/add-a-marker/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/add-image-animated/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/add-image-generated/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/add-image-missing-generated/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/add-image-stretchable/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/add-image/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/adjust-layer-opacity/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/animate-a-line/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/animate-camera-around-point/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/animate-images/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/animate-marker/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/animate-point-along-line/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/animate-point-along-route/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/attribution-position/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/camera-animation/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/canvas-source/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/center-on-symbol/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/change-building-color-based-on-zoom-level/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/change-case-of-labels/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/check-for-support/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/cluster-html/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/cluster/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/color-switcher/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/custom-marker-icons/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/custom-style-id/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/custom-style-layer/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/dancing-buildings/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/data-driven-circle-colors/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/data-driven-lines/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/data-join/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/disable-rotation/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/disable-scroll-zoom/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/display-and-style-rich-text-labels/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/drag-a-marker/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/drag-a-point/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/fallback-image/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/fill-pattern/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/filter-features-within-map-view/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/filter-markers-by-input/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/fitbounds/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/filter-markers/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/flyto-options/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/flyto/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/forward-geocode-custom-data/": {
    "product": "Mapbox GL JS,Geocoding API",
    "platform": "web",
    "service": "maps,search",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/fullscreen/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/game-controls/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/geojson-layer-in-stack/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/geojson-line/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/geojson-markers/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/heatmap-layer/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/geojson-polygon/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/hillshade/": {
    "product": "Mapbox GL JS,Mapbox Terrain-RGB tileset",
    "platform": "web",
    "service": "maps,data services",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/hover-styles/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/image-on-a-map/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/interactive-false/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/jump-to/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/language-switch/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/line-gradient/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/live-geojson/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/live-update-feature/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/locate-user/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/local-ideographs/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/map-tiles/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-compare/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-directions/": {
    "product": "Directions API,Mapbox GL JS",
    "platform": "web",
    "service": "navigation,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-draw/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-geocoder-accept-coordinates/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-geocoder-custom-render/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-geocoder-limit-region/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-geocoder-no-map/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-geocoder-outside-the-map/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-geocoder-with-flyto/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-geocoder-with-language/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-geocoder/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mapbox-gl-rtl-text/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/marker-from-geocode/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/measure/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/mouse-position/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/navigation/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/multiple-geometries/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/offset-vanishing-point-with-padding/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/playback-locations/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/polygon-popup-on-click/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/point-from-geocoder-result/": {
    "product": "Geocoding API,Mapbox GL JS",
    "platform": "web",
    "service": "search,maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/popup-on-hover/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/popup-on-click/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/popup/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/query-similar-features/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/queryrenderedfeatures-around-point/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/queryrenderedfeatures/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/render-world-copies/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/restrict-bounds/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/satellite-map/": {
    "product": "Mapbox GL JS,Mapbox Satellite tileset",
    "platform": "web",
    "service": "maps,data services",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/scroll-fly-to/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/set-popup/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/set-perspective/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/setstyle/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/simple-map/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/third-party/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/style-ocean-depth-data/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/timeline-animation/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/toggle-interaction-handlers/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/toggle-layers/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/toggle-worldviews/": {
    "product": "Mapbox GL JS,Mapbox Streets tileset",
    "platform": "web",
    "service": "maps,data services",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/updating-choropleth/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/using-box-queryrenderedfeatures/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/variable-label-placement/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/vector-source/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/visualize-population-density/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/video-on-a-map/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/zoomto-linestring/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/example/wms/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps",
    "content_type": "example"
  },
  "/mapbox-gl-js/style-spec/glyphs/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/layers/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/light/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/root/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/sprite/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/transition/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/types/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/expressions/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/other/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/style-spec/sources/": {
    "product": "Mapbox Style Specification",
    "platform": "",
    "service": "maps",
    "content_type": "specification"
  },
  "/mapbox-gl-js/plugins/": {
    "product": "Mapbox GL JS",
    "platform": "web",
    "service": "maps"
  }
}
```

**Pull request merged**